.github/workflows/test.lint_pr.yml

```diff
@@ -27,9 +27,11 @@ jobs:
           #  fix
           #  feat
           # Configure which scopes are allowed (newline delimited).
-          # Append a scope for each service here
           scopes: |
+            backend
+            webapp
```

Comment:
Why this in frontend/.github/workflows?

Reply:
?

Reply:
I don't understand this change! Didn't we have them and it came with this PR?